.github/workflows/python-app.yml

```diff
@@ -26,15 +26,15 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install flake8 pytest flit
+        curl -sSL https://install.python-poetry.org | python3 - --version 1.2.0b2
```

Comment:
Is there a reason to use 1.2.0b2 over 1.2.0b3 ? 
can we use 1.1.14 ? or did you have to update to 1.2 to solve the problem of installing two versions of wheel?

Reply:
So I had to update to 1.2.0b2 because of the problem of wheels unfortunately, I'll open an issue to make sure we don't forget when there's a version that"s not beta that goes live